docs/pull_request_template.md

```diff
@@ -1,22 +1,44 @@
+<!--
+- Copyright (c) 2019-2020, Arm Limited and Contributors
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
 ## Description
 
 Please include a summary of the change, new sample or fixed issue. Please also include relevant motivation and context.
 Please read the [contribution guidelines](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md)
 
-## Checklist:
+## General Checklist:
 
-Do not submit your PR without all of the below being checked:
+Please ensure the following points are checked:
 
 - [ ] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Code-Style)
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
-- [ ] I have tested my sample on at least one compliant Vulkan implementation
-- [ ] I have stated on what implementation the sample has been tested so that others can test on different implementations and platforms
-- [ ] If my sample is vendor-specific, I have [tagged it appropriately](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)
+- [ ] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Copyright-Notice-and-License-Template)
+- [ ] I have commented any added functions (in line with Doxygen)
+- [ ] I have commented any code that could be hard to understand
 - [ ] My changes do not add any new compiler warnings
-- [ ] Vulkan validation layer output is clean on at least one compliant implementation
-- [ ] Any dependent changes (e.g. assets) have been merged and published in downstream modules
+- [ ] My changes do not add any new validation layer errors or warnings
 - [ ] I have used existing framework/helper functions where possible
-- [ ] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Copyright-Notice-and-License-Template)
 - [ ] My changes build and run on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)
+
+## Sample Checklist
+
+If your PR contains a new or modified sample, these further checks must be carried out *in addition* to the General Checklist:
```

Comment:
Will including modified samples here cause confusion? If you fix a bug or typo you are effectively modifying the sample right?

Reply:
How can I rephrase this do you reckon? I wanted to be clear that this checklist should apply to modified samples rather than just new. Even if it is just a typo I think the checklist should be adhered to, for trivial changes it should be fairly straightforward to check them all.

Reply:
Ok, on that basis I think we can leave as is.